style/scss/_header.scss

```diff
@@ -375,5 +381,64 @@ header {
   background-color: $green-mint;
 }
 
-@media only screen and (max-width: 1025px) {
+@media screen and (min-width: $phone-size) and (max-width: $tablet-size) {
+    .nav {
+        &__links__list {
+            display: none;
+            height: calc(100vh - 60px);
+            padding-top: 10rem;
+            gap: 2rem;
+
+            & li > a{
+             font-size: 3rem;
+             font-weight: 500;
+            }
+    
+            & #socialLinks{
+                display: grid;
+                grid-template-columns: repeat(3, 1fr);
+                row-gap: 0.6rem;
+                column-gap: 1rem;
```

Comment:
Can you please try to make the gap between the icons bigger? Something like 2rem for rows and 3rem for columns.

Reply:
Added   row-gap: 2rem; column-gap: 3rem;